wires/plugins/highlights/plugin.py

```diff
@@ -113,13 +113,17 @@ async def on_message(event: hikari.GuildMessageCreateEvent) -> None:
     guild = unwrap(event.get_guild())
     for user, triggers in notifications.items():
         channel = await plugin.app.rest.create_dm_channel(user)
+        footer = guild.name
+        if (source_channel := event.get_channel()):
```

Comment:
`event.get_channel()` can be put under the `guild = unwrap(event.get_guild())` line, to avoid calling this method multiple times. The whole footer logic can, actually.

Reply:
Can `hikari.Embed` objects be reused for multiple messages?

Reply:
Actually yeah, the whole embed logic should be moved out of the loop. But was just thinking the footer string

Reply:
Actually yes, but I was just thinking footer string. The entire embed logic should probably be moved out the loop, but that's something for another PR.